src/constants/index.js

```diff
@@ -31,6 +31,9 @@ export const PROVISION_SOURCE_TEMPLATE = 'Template';
 
 export const POD_NETWORK = 'Pod Networking';
 
+export const ANNOTATION_PXE_INTERFACE = 'cnv.ui.pxeInterface';
```

Comment:
nitpick: can we order the lines alphabetically or at least group them by context (like `TEMPLATE_` or `ANNOTATION_`)?

Reply:
will do!

Reply:
done